### PR TITLE
Anonymous nuget list Test case

### LINKED ISSUE #1685: Test case for anonymous nuget list command
# Task Description
`s-w-i-t` has test case for `nuget list` command, it should be able to list packages without credentials.

# Task Relationships

This task:
* Relates to: https://github.com/strongbox/strongbox/issues/1662

# Useful Links

* https://github.com/strongbox/strongbox-web-integration-tests

# Help

* [Our chat](https://chat.carlspring.org/)
* Points of contact:
  * @carlspring
  * @fuss86
  * @sbespalov
  * @steve-todorov

### PR DESCRIPTION
**Pull request by anki2189**
This PR closes https://github.com/strongbox/strongbox/issues/1685

### CODE REVIEW

File: nuget/src/it/test-nuget-common-flow.groovy [hunk 1]
```diff
@@ -32,15 +32,15 @@ runCommand(targetPath, String.format(
     configPath))
 
 output = runCommand(targetPath, String.format(
-    "$nugetExec list Org.Carlspring -ConfigFile %s",
-    configPath))
+    "$nugetExec list Org.Carlspring -Source %s",
+    storageUrl))
```
Comment (steve-todorov) [inline nuget/src/it/test-nuget-common-flow.groovy:41]: This should probably be a separate command? I assume ` "$nugetExec list Org.Carlspring -ConfigFile %s"` is making the listing using credentials? Also, could you please check if `nuget` is not automatically picking up the `NuGet.config` file? (this file is created a few lines before this)

File: nuget/src/it/test-nuget-common-flow.groovy [hunk 2]
```diff
@@ -32,15 +32,15 @@ runCommand(targetPath, String.format(
...
 
 output = runCommand(targetPath, String.format(
     "$nugetExec delete Org.Carlspring.Strongbox.Examples.Nuget.Mono 1.0.0 -Source %s -NonInteractive -ConfigFile %s",
     storageUrl,
     configPath))
 output = runCommand(targetPath, String.format(
-    "$nugetExec list Org.Carlspring -ConfigFile %s",
-    configPath))
+    "$nugetExec list Org.Carlspring -Source %s",
+    storageUrl))
```
Comment (steve-todorov) [inline nuget/src/it/test-nuget-common-flow.groovy:55]: This should probably be a separate command?

Comment (anki2189) [reply nuget/src/it/test-nuget-common-flow.groovy:55]: Changed to separate command

File: nuget/src/it/test-nuget-common-flow.groovy (hunk 1 above)
Comment (anki2189) [reply nuget/src/it/test-nuget-common-flow.groovy:41]: Changed to separate command.
Tested it with only `"$nugetExec list Org.Carlspring"` it was failing with reason 
**Unable to load the service index for source https://api.nuget.org/v3/index.json**
Hence we can say, it is not picking `NuGet.config`

**Review by steve-todorov — APPROVED** [review]
LGTM :)

**Pull request merged**